packages/@magic-sdk/types/src/modules/auth-types.ts

```diff
@@ -146,6 +146,10 @@ export enum AuthEventOnReceived {
   IDTokenCreated = 'Auth/id-token-created',
 }
 
+export enum FarcasterLoginEventEmit {
+  SuccessSignIn = 'success_sign_in',
```

Comment:
Could we turn this to `farcaster/success_sign_in`

Reply:
[866a300](https://github.com/magiclabs/magic-js/pull/750/commits/866a30096b897e1d15e4f5c3ef25a2d5b22deb9a)